config/tests/test_urls.py

```diff
@@ -0,0 +1,7 @@
+from socialhome.tests.utils import SocialhomeTestCase
+
+
+class TestSchemaView(SocialhomeTestCase):
+    def test_page_renders(self):
+        response = self.client.get("/api/")
```

Comment:
Would prefer `reverse("api-docs")` instead of hardcoded url - if the route is changed in the URL configuration, then test wont need to be updated.

Also, the following is not mandatory to change, but as a tip. `SocialhomeTestCase` bases of `test_plus.TestCase` (https://django-test-plus.readthedocs.io/en/latest/), so you can shorten these two lines as follows:

```
self.get("api-docs")
self.response_200()
```

It's a great library for writing view tests :) And no need for reverse since test plus does that internally.